esmvalcore/_task.py

```diff
@@ -24,6 +25,9 @@
     'mip',
 }
 
+CMIP6_CITATION_URL = 'https://cera-www.dkrz.de/WDCC/ui/cerasearch/' \
+                     'cmip6?input=CMIP6.CMIP.'
```

Comment:
```suggestion
                     'cmip6?input=CMIP6.'
```
`CMIP` is the `activity_id`, so should not be hardcoded, or this will not work for other activities than CMIP